azurerm/resource_arm_automation_schedule.go

```diff
@@ -0,0 +1,199 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/arm/automation"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func resourceArmAutomationSchedule() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmAutomationScheduleCreateUpdate,
+		Read:   resourceArmAutomationScheduleRead,
+		Update: resourceArmAutomationScheduleCreateUpdate,
+		Delete: resourceArmAutomationScheduleDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"account_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			"resource_group_name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"start_time": {
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateStartTime,
+			},
+
+			"expiry_time": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
should this be `Computed: true`? (to read the value from Azure if it's not set locally)

Reply:
Fixed!